public/app/features/explore/hooks/useStateSync/index.ts

```diff
@@ -248,8 +255,9 @@ export function useStateSync(params: ExploreQueryParams) {
               acc[key] = undefined;
               return acc;
             }, {}),
-            panes: JSON.stringify(newParams.panes),
+            // we set the schemaVersion as the first parameter so that when URLs are truncated the schemaVersion is more likely to be present.
```

Comment:
Doesn't seem to work every time 🤔 


https://github.com/grafana/grafana/assets/745532/178fce75-4178-4458-8304-7da34b689cc4

Reply:
amazing catch! 
this happens when the query completes before Explore has done initializing (?)

i've pushed a tentative fix in https://github.com/grafana/grafana/pull/77784/commits/959a83a3c2c627e2306b3128524d93512f81eebf